rmbproxy/twin.go

```diff
@@ -0,0 +1,103 @@
+package rmbproxy
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/rs/zerolog/log"
+	"github.com/threefoldtech/zos/pkg/substrate"
+)
+
+func submitURL(timeIP string) string {
```

Comment:
convert to another name instead of the imperative form, maybe `formatZbusURL` and change timeIP to twinID